lib/renderer/chrome-api.ts

```diff
@@ -92,12 +107,14 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
       })
     },
 
+    // https://developer.chrome.com/extensions/runtime#method-getManifest
```

Comment:
TypeScript will associate these comments with the function if using the Docs style comments (e.g. `/** comment */`).
https://typedoc.org/guides/doccomments/

Reply:
That's actually on purpose – one shouldn't mistake the link for the _actual_ implementation, but it's a useful guide for developers coming in and working with this code.